lib/parallel-fields-extractor.js

```diff
@@ -0,0 +1,14 @@
+module.exports = (elasticSearchResponse) => {
+  const updatedHits = elasticSearchResponse.hits.hits.map((bib) => {
+    if (bib._source.parallelDisplayField) {
+      for (let field of bib._source.parallelDisplayField) {
+        const capitalizedFieldName = field.fieldName.charAt(0).toUpperCase() + field.fieldName.slice(1)
+        bib._source[`parallel${capitalizedFieldName}`] = [field['@value']]
```

Comment:
We need to check the `.index` vaue. If `field.index` is `3`, we actually want the value to be stored at index 3. So something like
```bib._source[`parallel${capitalizedFieldName}`][field.index] = field['@value']``` (after initializing as an array if it's not already initialized)